src/SnackSession.js

```diff
@@ -292,6 +300,94 @@ export default class SnackSession {
     return url;
   };
 
+  /**
+   * Generates a default app.json template for the snack.
+   * @returns {Object} the app.json template
+   * @function
+   */
+  generateAppJson = (): Object => {
+    const appJsonTemplate = {
+      expo: {
+        description: this.description,
+        slug: 'snack-' + this.snackId,
+        privacy: 'unlisted',
+        sdkVersion: this.sdkVersion,
+        version: '1.0.0',
+        orientation: 'portrait',
+        primaryColor: '#cccccc',
+        icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+        loading: {
+          icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+          hideExponentText: false,
+        },
+        packagerOpts: {
+          assetExts: ['ttf', 'mp4', 'otf'],
+        },
+        ios: {
+          supportsTablet: true,
+        },
+      },
+    };
+    return appJsonTemplate;
+  };
+
+  /**
+   * Builds an apk from the snack and returns a url to download the apk.
+   * @param 
+   * @returns {Promise.<void>} A promise that contains the url when fulfilled.
+   * @function
+   */
+  getADKUrlAsync = async (
+    appJson : Object,
+    opts: {
+      current?: boolean,
+      mode?: string,
+      platform?: string,
+      expIds?: Array<string>,
+      type?: string,
+      releaseChannel?: string,
+      bundleIdentifier?: string,
+      publicUrl?: string,
+      sdkVersion?: string,
+    } = {}
+  ): Promise<string> => {
+    opts.sdkVersion = this.sdkVersion;
+    opts.platform = 'android';
+    // call out to build api here with url
+    const { id: buildId } = await buildAdkAsync(appJson, opts);
+    const completedJob = await this.wait(buildId, appJson, {});
+    const artifactUrl = completedJob.artifactId
+        ? `https://expo.io/artifacts/${completedJob.artifactId}`
+        : completedJob.artifacts.url;
+
+    return artifactUrl;
+  };
+
+  async wait(buildId, appJson, { timeout = 1200, interval = 60 } = {}) {
```

Comment:
Should give wait a more descriptive name & have it end in async.  Prefix things that aren't intended to be called directly w/ _

My expectation would be that timeout / interval would be specified in milliseconds